core/cmd/cosmos_transaction_commands_test.go

```diff
@@ -0,0 +1,156 @@
+//go:build integration
+
+package cmd_test
+
+import (
+	"flag"
+	"strconv"
+	"testing"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/urfave/cli"
+
+	cosmosclient "github.com/smartcontractkit/chainlink-terra/pkg/cosmos/client"
+	cosmosdb "github.com/smartcontractkit/chainlink-terra/pkg/cosmos/db"
+
+	"github.com/smartcontractkit/chainlink/core/chains/cosmos/cosmostxm"
+	"github.com/smartcontractkit/chainlink/core/chains/cosmos/denom"
+	"github.com/smartcontractkit/chainlink/core/cmd"
+	"github.com/smartcontractkit/chainlink/core/internal/cltest"
+	"github.com/smartcontractkit/chainlink/core/internal/testutils"
+	"github.com/smartcontractkit/chainlink/core/internal/testutils/cosmostest"
+	"github.com/smartcontractkit/chainlink/core/internal/testutils/pgtest"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/services/keystore/keys/cosmoskey"
+)
+
+func TestClient_SendCosmosCoins(t *testing.T) {
+	t.Skip("requires cosmosd")
+	chainID := cosmostest.RandomChainID()
+	accounts, _, tendermintURL := cosmosclient.SetupLocalCosmosNode(t, chainID)
```

Comment:
Ah right, this method needs renaming on the relay side, it's still called `SetupLocalTerraNode`